test/moonwall.config.json

```diff
@@ -346,6 +346,54 @@
                     "endpoints": ["ws://127.0.0.1:8000"]
                 }
             ]
+        },
+        {
+            "name": "chopsticks_flashbox_upgrade",
+            "testFileDir": ["suites/rt-upgrade-chopsticks"],
+            "foundation": {
+                "type": "chopsticks",
+                "rtUpgradePath": "../target/release/wbuild/flashbox-runtime/flashbox_runtime.compact.compressed.wasm",
+                "launchSpec": [
+                    {
+                        "name": "db",
+                        "type": "parachain",
+                        "configPath": "./configs/stagenet-dancebox.yml"
```

Comment:
@fgamundi you need to create a new yml for flashbox, identical to the stagenet one, but with this url:

wss://fraa-flashbox-rpc.a.stagenet.tanssi.network